grails-app/jobs/com/causecode/fileuploader/DailyJob.groovy

```diff
@@ -20,17 +22,23 @@ class DailyJob {
     }
 
     def execute() {
-        log.info 'Started executing DailyJob..'
-        fileUploaderService.renewTemporaryURL()
-        fileUploaderService.moveFailedFilesToCDN()
+
+        boolean renewJobDisabled = Holders.config.jobs.fileUploader.renewURLs.disable ?: false
+
+        if (!renewJobDisabled) {
+            log.info 'Started executing DailyJob..'
+
+            fileUploaderService.renewTemporaryURL()
+            fileUploaderService.moveFailedFilesToCDN()
+
+            log.info 'Finished executing DailyJob.'
+        }
```

Comment:
Add else statement and log that the job has been disabled in installing app.

Reply:
Ok